includes/templates/template_default/templates/tpl_zc_install_suggested_default.php

```diff
@@ -14,7 +14,7 @@
 $docsPath = (file_exists('docs/index.html')) ? 'docs/index.html' : (file_exists('../docs/index.html') ? '../docs/index.html' : '');
 ?>
 <!DOCTYPE html>
-<html <?php echo defined('HTML_PARAMS') ? HTML_PARAMS : '';?>>
```

Comment:
Instead of removing, why not just use the default `dir="ltr" lang="en"` since the hard-coded language is English?